main.tf

```diff
@@ -1,16 +1,23 @@
 provider "packet" {
-  auth_token = "${var.auth_token}"
+  auth_token = var.auth_token
 }
 
 resource "packet_project" "kubenet" {
+  organization_id = var.organization_id
+  
   name = "k8s-bgp"
```

Comment:
Can we please make the `project_name` as a variable, otherwise it is hard to create a project per team or environment (e.g. dev, prod)?

Reply:
@tabdulazim great idea, i changed the project name to a variable.